### ISSUE TITLE
Expose ES tag keys directly in the OTEL config file

### ISSUE THREAD

**Issue by pavolloffay** [jaegertracing/jaeger#2301]
Created from https://github.com/jaegertracing/jaeger/pull/2295#discussion_r442871570

Add tag keys directly in the OTEL config file instead of a separate tag key file.

```
      --es.tags-as-fields.config-file string                  (experimental) Optional path to a file containing tag keys which will be stored as object fields. Each key should be on a separate line.

```

**Comment by joe-elliott** [comment]
@pavolloffay I'd like to work on this feature.  

This is marked `area/otel`, but I assume this would require changes in the elastic search options, opentelemetry translator and the traditional jaeger es spanstore.

**Comment by pavolloffay** [comment]
The Config/Options objects will have to be adjusted. The translator and spanstore shouldn't require any changes since they should already get parsed keys.

**Comment by joe-elliott** [comment]
Ok, that's what I was seeing.  Just wanted to make sure this wasn't an otel change only for some reason.

**Comment by joe-elliott** [comment]
@pavolloffay this was fixed in #2417.  I mentioned the wrong # in the PR.  Apologies for the mixup.

**Comment by pavolloffay** [comment]
Done in https://github.com/jaegertracing/jaeger/pull/2417
